jenkins/security/whitesource_container_scan.groovy

```diff
@@ -0,0 +1,202 @@
+                }
+            }
+        }
+
+        stage ('Setup K8s Cluster') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Setup K8s Cluster', script: '''
+                    pushd solutions/kubernetes/
+                        echo $hosts | tr ' ' '\n' > hosts
+                        cat hosts
+                        if [ "$(cat hosts | wc -l)" -eq 2 ]
+                        then
+                           echo "Current configuration does not support 2 node CORTX cluster deployment. Please try with 1 or more than two nodes."
+                           echo "Exiting Jenkins job."
+                           exit 1
+                        fi
+                        ./cluster-setup.sh ${PODS_ON_PRIMARY}
+                    popd
+                '''
+            }
+        }
+
+        stage ('Deploy CORTX') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Deploy CORTX Components', script: '''
+                    pushd solutions/kubernetes/
+                        export SOLUTION_CONFIG_TYPE=automated
+                        export CORTX_SCRIPTS_BRANCH=${CORTX_SCRIPTS_BRANCH}
+                        export CORTX_SCRIPTS_REPO=${CORTX_SCRIPTS_REPO}
+                        export CORTX_ALL_IMAGE=${CORTX_ALL_IMAGE}
+                        export CORTX_SERVER_IMAGE=${CORTX_SERVER_IMAGE}
+                        export CORTX_DATA_IMAGE=${CORTX_DATA_IMAGE}
+                        export DEPLOYMENT_METHOD=${DEPLOYMENT_METHOD}
+                        export SNS_CONFIG=${SNS_CONFIG}
+                        export DIX_CONFIG=${DIX_CONFIG}
+                        export EXTERNAL_EXPOSURE_SERVICE=${EXTERNAL_EXPOSURE_SERVICE}
+                        export CONTROL_EXTERNAL_NODEPORT=${CONTROL_EXTERNAL_NODEPORT}
+                        export S3_EXTERNAL_HTTP_NODEPORT=${S3_EXTERNAL_HTTP_NODEPORT}
+                        export S3_EXTERNAL_HTTPS_NODEPORT=${S3_EXTERNAL_HTTPS_NODEPORT}
+                        ./cortx-deploy.sh --cortx-cluster
+                    popd
+                '''
+            }
+        }
+
+        stage ('WhiteSource container scan') {
+           steps {
+                sh label: 'WhiteSource container scanning', script: '''
+                    pushd script/security/
+                        export SOLUTION_CONFIG_TYPE=automated
+                        export WHITESOURCE_SERVER_URL=${WHITESOURCE_SERVER_URL}
+                        export API_KEY=${API_KEY}
+                        export USER_KEY=${USER_KEY}
+                        export PRODUCT_NAME=${PRODUCT_NAME}
+                        export DOCKER_REGISTRY=${DOCKER_REGISTRY}
+                        export MAIN_POD=${MAIN_POD}
+                        export WORKER_POD=${WORKER_POD}
+                        export PULL_SECRET=${PULL_SECRET}
+                        ./whitesource_container_scan.sh
```

Comment:
Do we need any change in ./whitesource_container_scan.sh. If so please add to this PR only.

Reply:
Ok